docs/2_3_common_states.adoc

```diff
@@ -329,17 +329,17 @@ In <<InitializationMode>>, the FMU computes initial values at the start time lat
 |Set variables latexmath:[\mathbf{v}_{\mathit{initial=exact}}]
 |<<get-and-set-variable-values,`fmi3Set{VariableType}`>>
 
-|Set continuous-time and discrete-time <<input,`inputs`>>  latexmath:[\mathbf{u}_{c+d}(\mathbf{t}_{start})]
+|Set continuous-time and discrete-time <<input,`inputs`>>  latexmath:[\mathbf{u}_{c+d}(t_{start})]
 |<<get-and-set-variable-values,`fmi3Set{VariableType}`>>
 
-|latexmath:[(\mathbf{y}_{c+d}, \mathbf{\dot{x}}_c, \mathbf{x}_{c+d}, ^{\bullet}\mathbf{x}_d, \mathbf{z}, \mathbf{r}, \mathbf{w}_{c+d}, \mathbf{b}, \mathbf{T}_{\mathit{shift}}, \mathbf{T}_\mathit{interval}) := \mathbf{f}_{\mathit{init}}(\mathbf{u}_{c+d}, \mathbf{p}, \mathbf{t}_{start}, \mathbf{v}_{\mathit{initial=exact}})]
+|latexmath:[(\mathbf{y}_{c+d}, \mathbf{\dot{x}}_c, \mathbf{x}_{c+d}, ^{\bullet}\mathbf{x}_d, \mathbf{z}, \mathbf{r}, \mathbf{w}_{c+d}, \mathbf{b}, \mathbf{T}_{\mathit{shift}}, \mathbf{T}_\mathit{interval}) := \mathbf{f}_{\mathit{init}}(\mathbf{u}_{c+d}, \mathbf{p}, t_{start}, \mathbf{v}_{\mathit{initial=exact}})]
```

Comment:
\mathbf{T}_{\mathit{shift}} and \mathbf{T}_\mathit{interval}) ?

Reply:
Same as above: I have unboldet all T and \mathit-ed all subscripts with multiple characters to have a nicer (tighter) typesetting compared to the subscript without. Please unresolve if somethings is still amiss - or add another comment like this one with a pointer to the issue.